src/Spice86/Startup.cs

```diff
@@ -1,54 +1,80 @@
-namespace Spice86; 
+namespace Spice86;
 
-using Microsoft.Extensions.DependencyInjection;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.ApplicationLifetimes;
+
+using OxyPlot.Avalonia;
 
 using Serilog.Events;
 
 using Spice86.Core.CLI;
-using Spice86.DependencyInjection;
+using Spice86.Core.Emulator;
 using Spice86.Shared.Interfaces;
-/// <summary>
-/// Provides a method to initialize services and set the logging level based on command line arguments.
-/// </summary>
-internal static class Startup {
+
+public class Startup {
+    private readonly ILoggerService _loggerService;
     
-    /// <summary>
-    /// Initializes the service collection and sets the logging level.
-    /// </summary>
-    /// <param name="commandLineArgs">The command line arguments.</param>
-    /// <returns>A <see cref="ServiceProvider"/> instance that can be used to retrieve registered services.</returns>
-    public static ServiceProvider StartupInjectedServices(string[] commandLineArgs)
-    {
-        ServiceCollection services = new ServiceCollection();
-        services.AddLogging();
-        ServiceProvider serviceProvider = services.BuildServiceProvider();
-        SetLoggingLevel(serviceProvider, commandLineArgs);
-        return serviceProvider;
+    
+    public Startup(ILoggerService loggerService) {
+        _loggerService = loggerService;
     }
-
-    /// <summary>
-    /// Sets the logging level based on the command line arguments.
-    /// </summary>
-    /// <param name="serviceProvider">The <see cref="ServiceProvider"/> instance used to retrieve the <see cref="ILoggerService"/>.</param>
-    /// <param name="commandLineArgs">The command line arguments.</param>
-    private static void SetLoggingLevel(ServiceProvider serviceProvider, string[] commandLineArgs) {
-        ILoggerService? loggerService = serviceProvider.GetService<ILoggerService>();
-        if (loggerService is null) {
-            return;
+    
+    public void StartApp(string[] args) {
```

Comment:
I think we'd better inject the configuration here, rather than the args.

Reply:
Done.